src/tools/sdk/_windows.md

```diff
@@ -1,23 +1,41 @@
+###How to install?
+
 You can install the Dart SDK using [Chocolatey.][Chocolatey]
 
-{{site.alert.important}}
-These commands require administrator privileges.
-If you need help on starting an administrator-level command prompt,
-try a search like
-<em><a href="https://www.google.com/search?q=cmd+admin"
-  target="blank">cmd admin</a>.</em>
-{{site.alert.end}}
+> **_NOTE:_** These commands require administrative privileges. To start command prompt as an administrator following these steps:
+> 
+> - Press Windows+R to open the “Run” box. 
+> - Type “cmd” into the box 
+> - Press Ctrl+Shift+Enter to run the command as an administrator.
 
-To install the Dart SDK:
+####To install the Dart SDK:
 
 ```terminal
 C:\> choco install dart-sdk
 ```
 
-To **upgrade** the Dart SDK:
+#####To upgrade the Dart SDK:
 
 ```terminal
 C:\> choco upgrade dart-sdk
 ```
 
-[Chocolatey]: https://chocolatey.org
+####Default installation directory:
+
+```terminal
+C:\tools\dart-sdk
+```
```

Comment:
Where did you find this? I don't have a Windows PC at the moment to try this, but I found on [Chocolatey's FAQ](https://docs.chocolatey.org/en-us/faqs#where-does-chocolatey-install-by-default) that it by default installs at:

> As of version 0.9.8.24, binaries, libraries and Chocolatey components install in C:\ProgramData\chocolatey (environment variable %ProgramData%) by default.

Reply:
Yes, that's the default path but not every package is installed there. In the same link, read under the heading: "Why doesn't a package install software to Program Files?". 

Apparently this is why everyone was having difficulty finding since everyone was trying to look inside program data/files.

I tested on my PC and it installed inside the tools folder.